core/src/dird/ua_output.cc

```diff
@@ -498,14 +498,132 @@ static void SetQueryRange(PoolMem& query_range, UaContext* ua, JobDbRecord* jr)
   }
 }
 
+struct ListCmdOptions {
+  bool count;
```

Comment:
"count" is a little misleading, may find another word that fits more.

Reply:
I think count is fine, as it exactly reflects the command line argument, which may or may not be given by user input.